rtic/src/export/riscv_esp32c3.rs

```diff
@@ -0,0 +1,105 @@
+use esp32c3::INTERRUPT_CORE0; //priority threshold control
+pub use esp32c3::{Interrupt, Peripherals};
```

Comment:
I see you set your core peripherals as the PAC's peripherals. So this is basically what we have done. Thus, the core peripherals and device peripherals will be exactly the same. This is why I wonder if the core peripherals are needed for RISC-V targets, as there is no such thing. Maybe we can make it optional and include it only for Cortex. However, I think @korken89 would rather not modify struct fields depending on the backend.

Reply:
I don't think that API is set in stone - if there is a good reason to deviate I see no reason to strictly follow the old one :)